Makefile

```diff
@@ -131,7 +131,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: ## Build docker image with the manager.
-	$(CONTAINER_TOOL) build -t ${IMG} .
+	$(CONTAINER_TOOL) build --platform=linux/amd64 -t ${IMG} .
```

Comment:
Not sure if this would break anything? I guess we don't rely on Makefile in our build and docker image release process so it's fine.

Reply:
for example, we publish arm images today.

Reply:
Whoops, I should revert this